Exiled.Events/Patches/Generic/Scp173BeingLooked.cs

```diff
@@ -22,12 +29,141 @@ namespace Exiled.Events.Patches.Generic
 
     using static HarmonyLib.AccessTools;
 
+    ///// <summary>
```

Comment:
Why not delete these comments?

Reply:
Fair, I just wanted someone to have the prefix to compare against